packages/anoncreds/src/AnonCredsModuleConfig.ts

```diff
@@ -1,10 +1,47 @@
 import type { AnonCredsRegistry } from './services'
+import type { Anoncreds } from '@hyperledger/anoncreds-shared'
 
 /**
  * @public
  * AnonCredsModuleConfigOptions defines the interface for the options of the AnonCredsModuleConfig class.
  */
 export interface AnonCredsModuleConfigOptions {
+  /**
```

Comment:
The anoncreds dependency should actually be added to the AnonCredsRs module (I can understand this is confusing). The `@aries-framework/anoncreds` (so no `-rs`) can be used with multiple anoncreds implementations (either `anoncreds-rs` or `indy-sdk`)

Reply:
You added the `anoncreds` property to the `@aries-framework/anoncreds` package (`AnonCredsModule`), while it needs to be added to the `@aries-framework/anoncreds-rs` (`AnonCredsRsModule`)